_join/index.md

```diff
@@ -47,29 +47,27 @@ We're building a team that reflects the United States. We don't discriminate bas
 
 ### How we collect and review applications
 
-We collect applications during the period specified in the job posting (usually five days to two weeks). We consider all applications collected within a given period a cohort and evaluate them against each other.
+We collect applications during the period specified in the job posting (usually five business days). We consider all applications collected within a given period a cohort and evaluate them against each other.
 
-Once the application period ends, you can't add to or modify your application.
+Once the application collection period ends, you can't add to or modify your application.
 
-We begin evaluating applications using a qualification process called [Category Rating](https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/#url=Category-Rating). Following this process, we evaluate each application against specific competencies (you can find these in the job posting) and categorize each as Superior Qualified, Best Qualified, Well Qualified, Qualified, or Not Qualified.
-
-Once we've evaluated all applications within the cohort, we send all the applications and evaluation scores to General Services Administration (GSA) Human Resources, our partner in the hiring process, to apply [Veterans' Preference](http://www.fedshirevets.gov/job/vetpref/index.aspx) for candidates who claim it. GSA HR returns a list of candidates who were categorized as Superior Qualified, and the selection panel (usually the hiring manager) then identifies which candidates to interview.
+A panel of Subject Matter Experts (SMEs) will review each application. The top qualified candidates will be contacted to continue on to the interview process. 
```

Comment:
The language on the new section if fine, but I'm curious why we're removing a lot of information in favor of this new sentence. If this info is no longer correct, can we give a more detailed version of the current process?